src/main/kotlin/apply/application/MissionDtos.kt

```diff
@@ -25,19 +28,22 @@ data class MissionData(
     @field:NotBlank
     var description: String = "",
 
+    var judgmentItemData: JudgmentItemData = JudgmentItemData(),
```

Comment:
r: 애너테이션이 없으니 붙여 쓸 수 있겠네요.

```suggestion
    var description: String = "",
    var judgmentItemData: JudgmentItemData = JudgmentItemData(),
```